viperx/validate.go

```diff
@@ -52,6 +54,7 @@ func Validate(name string, content []byte) error {
 	if err := json.NewEncoder(&b).Encode(viper.AllSettings()); err != nil {
 		return errors.WithStack(err)
 	}
+	l.WithFields(viper.AllSettings()).Debug("detected config values")
```

Comment:
This is a pretty big breaking change for one debug statement, does that really make sense?